book/src/development/infrastructure/sync.md

```diff
@@ -15,107 +15,82 @@ done in a bi-weekly basis if there's no urgent changes. This is done starting on
 the day of the Rust stable release and then every other week. That way we
 guarantee that we keep this repo up to date with the latest compiler API, and
 every feature in Clippy is available for 2 weeks in nightly, before it can get
-to beta. For reference, the first sync following this cadence was performed the
+to beta. For reference, the first sync following this cadence was performed on
 2020-08-27.
 
-This process is described in detail in the following sections. For general
-information about `subtree`s in the Rust repository see [the rustc-dev-guide][subtree].
+This process is described in detail in the following sections.
 
-## Patching git-subtree to work with big repos
+## Installing `josh-proxy`
 
-Currently, there's a bug in `git-subtree` that prevents it from working properly
-with the [`rust-lang/rust`] repo. There's an open PR to fix that, but it's
-stale. Before continuing with the following steps, we need to manually apply
-that fix to our local copy of `git-subtree`.
+The sync is done with [JOSH] and fully scripted with `cargo dev sync`. The only
+requirement is to install the `josh-proxy` binary from GitHub
 
-You can get the patched version of `git-subtree` from [here][gitgitgadget-pr].
-Put this file under `/usr/lib/git-core` (making a backup of the previous file)
-and make sure it has the proper permissions:
+<!-- FIXME: Change to a release version once the fix is merged -->
```

Comment:
If we merge it like this, can you replace `the fix` with an actual issue? Something like josh#1728917293

Reply:
Oh, this was already merged. But there wasn't a Josh release since then. I will update this comment.